File: file_chua_api/donation/donation.php
<?php
class Donation {
    private $id ; 
    private $amount ; 
    private $paymenttype ; 
    private $upvotes ; 
    function Donation($id, $amount , $paymenttype, $upvotes){
        $this->_id = $id;
        $this->amount = $amount;
        $this->paymenttype = $paymenttype;
        $this->upvotes = $upvotes;
    }
  
}

?>

File: app/src/main/java/ie/app/models/Donation.java
package ie.app.models;

import androidx.annotation.NonNull;

public class Donation {
    public String _id;
    public int amount;
    public String paymenttype;
    public int upvotes;

    public Donation(int amount, String method, int upvotes) {
        this.amount = amount;
        this.paymenttype = method;
        this.upvotes = upvotes;
    }

    public Donation() {
        this.amount = 0;
        this.paymenttype = "";
        this.upvotes = 0;
    }

    @NonNull
    public String toString()
    {
        return _id + ", " + amount + ", " + paymenttype + ", " + upvotes;
    }
}
